ext-src/packages/npm/NpmUtils.ts

```diff
@@ -13,47 +13,166 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import * as path from "path";
+import * as path from 'path';
 import * as fs from "fs";
 import * as _ from "lodash";
 
 import { exec } from "../../utils/exec";
 import { PackageDependenciesHelper } from "../PackageDependenciesHelper";
 import { NpmPackage } from "./NpmPackage";
+import { NPM_SHRINKWRAP_JSON, YARN_LOCK, PACKAGE_LOCK_JSON } from "./NpmScanType";
+
+const NPM_SHRINKWRAP_COMMAND = `npm shrinkwrap`;
```

Comment:
trivial: i think back ticks may be mostly for template literals, maybe single quotes for this and next two lines instead

Reply:
In: 32cd19b